data/mods/ssb/abilities.ts

```diff
@@ -662,6 +662,32 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		},
 	},
 
+	// Nolali
+	burningsoul: {
+		shortDesc: "Drought + Sturdy.",
+		onStart(source) {
+			for (const action of this.queue) {
+				if (action.choice === 'runPrimal' && action.pokemon === source && source.species.id === 'groudon') return;
+				if (action.choice !== 'runSwitch' && action.choice !== 'runPrimal') break;
+			}
```

Comment:
this thing cant go primal, and theres no groudon. dw bout this line

Reply:
someone smack me if someone decides to go skill swap red orb groudon for whatever reason